jsapp/js/components/submissions/submissionUtils.ts

```diff
@@ -338,12 +388,12 @@ function getRowData(name, survey, data) {
 
 /**
  * Tells if given row is an immediate child of given group
- * @param {string} rowName
- * @param {string|null} groupPath - null for root level rows
- * @param {Array<object>} survey
- * @returns {boolean}
  */
-function isRowFromCurrentGroupLevel(rowName, groupPath, survey) {
+function isRowFromCurrentGroupLevel(
+  rowName: string,
+  /** Null for root level rows. */
+  groupPath: string|null,
+  survey: SurveyRow[]) {
```

Comment:
Pretty trivial but is this `)` supposed to be there (instead of on the next line) or a missed linting?

Reply:
Good catch, supposed to be on new line :)